docs/contacts/preference_center.rst

```diff
@@ -4,5 +4,192 @@ Preference center
 #################
 .. vale on
 
+.. vale off
+
+Manage Contact Preferences
+**************************
+
+.. vale on
+
+When managing a Contact you can set the Contacts preference of communication. You can access the Contact's preference center when viewing a Contact's profile. From the dropdown menu click the preference menu. A new modal window should appear with a tab to set the preferred Channels and frequency of Contact, as well as the option to pause communication within a given period of time. 
+The second tab gives the option to add or remove the Contact from global Categories used in Emails or Categories. 
+The third tab allowS to add or remove the Contact from Segments it belongs to.
+
+.. vale off
+
+Preferred Channels and frequency
+********************************
+
+.. vale on
+
+.. image:: images/preferences.png
+    :align: center
+    :alt: Screenshot of Preference
+
+|
+
+In this window you can switch Channels of communication, set the frequency of the communication via each Channel enabled, and set one of the Channels as a preferred Channel.
+
+To set a Channel as a don't contact me through this Channel, un-tick the tick-box next to the Channel name in the first column.
+
+When a Channel is selected, these will be used to send marketing messages if there is a message set for any of the Channels selected. You can also set the frequency of the communication, as in this example the frequency is set to "Send me emails twice a day" but pause them during the "``1st`` of November 2022 to the ``30th`` of November 2022" . Email is also set as the preferred Channel, so if the same message is set for both email and ``sms``, it will only send the Email version of the message to the selected Contact.
```

Comment:
```suggestion
When a Channel is selected, these will be used to send Marketing Messages if there is a message set for any of the Channels selected. You can also set the frequency of the communication, as in this example the frequency is set to "Send me emails twice a day" but pause them during the "``1st`` of November 2022 to the ``30th`` of November 2022" . Email is also set as the preferred Channel, so if the same message is set for both Email and ``sms``, it will only send the Email version of the message to the selected Contact.
```